Porting/release_managers_guide.pod

```diff
@@ -197,7 +197,7 @@ The work of building a release candidate for an even numbered release
 release candidate.  Some of the following steps should be done regularly,
 but all I<must> be done in the run up to a release.
 
-=head3 dual-life CPAN module synchronisation
+=head3 Dual-Life CPAN module synchronisation
```

Comment:
I don't think "Life" needs to be capitalized here.

Reply:
Can a maintainer amend my commit for this? I tried, and I'm afraid to mess up git history by pushing 🙂

Reply:
Resolved in https://github.com/Perl/perl5/pull/21463